### PR TITLE
Add yelp button to show page

### PR DESCRIPTION
**Pull request by zachjamesgreen**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Build is passing
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):
## What is the current behavior?

Ther is now a button a user can click to take them to the yelp search page
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/actions/auth_actions/confirm_login_action.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ Actions.register(CONFIRM_LOG_IN, payload => {
   })
   .then(checkStatus)
   .then(response => {
-    authStore.setAuthed(response.data.authToken);
+    authStore.setAuthed(response.data.token);
```
Comment (mkrumholz) [inline src/actions/auth_actions/confirm_login_action.js:15]: Does this relate to changing how we do auth here? Trying to follow this piece, but not sure why the change from authToken

**Review by mkrumholz — CHANGES_REQUESTED** [review]
Looks like we've got a merge conflict with the stars, but otherwise, excited to try and get this connection up and running

**Review by markcyen — APPROVED** [review]
Look good to me... just that one question Molly asked.

**Review by mkrumholz — APPROVED** [review]
